openquake/engine/engine.py

```diff
@@ -335,8 +335,9 @@ def run_calc(job_id, oqparam, exports, hazard_calculation_id=None, **kw):
         return
     try:
         if OQ_DISTRIBUTE.endswith('pool'):
+            # Report the number of "visible" cores, not the total system cores
             logs.LOG.warning('Using %d cores on %s',
-                             parallel.Starmap.num_cores, platform.node())
+                             len(os.sched_getaffinity(0)), platform.node())
```

Comment:
It would be better to fix the cpu count in https://github.com/gem/oq-engine/blob/master/openquake/baselib/parallel.py#L595 directly (maybe using `psutil` as discussed) and not here, but @micheles has the final word on this

Reply:
Oh great, there it is!
I can fix it properly using psutil, if you find this useful. In case: should I simply revert the commit, rebase my branch or create a new PR?

Reply:
Just add a new commit that changes the right line and restore the currently changed line as it was before.

Reply:
Done!